src/nvim/log.h

```diff
@@ -4,6 +4,8 @@
 #include <stdio.h>
 #include <stdbool.h>
 
+#include "nvim/os/os_defs.h"
```

Comment:
@sethjackson, regarding https://connect.microsoft.com/VisualStudio/feedback/details/1729717. Just upgraded my VS to 2015 Update 1 (https://www.visualstudio.com/en-us/news/vs2015-update1-vs.aspx) and It is fixed: compiles both with /TC and /TP. :)
AppVeyor will be updated subsequently https://github.com/appveyor/ci/issues/526.